### PR TITLE
Announce Tenzir Platform v1.5

### PR DESCRIPTION
**Pull request by dit7ya**
This PR adds a blogpost for the upcoming v1.5 release of the Tenzir Platform.

### CODE REVIEW

File: web/blog/tenzir-platform-v1.5/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Tenzir Platform v1.5: Multi-node Dashboards, Contexts, Example Pipelines, and more"
+slug: tenzir-platform-v1.4
```
Comment (lava) [inline web/blog/tenzir-platform-v1.5/index.md:3]: v1.5

**Review by dominiklohmann — APPROVED** [review]
As discussed I pushed my review feedback directly on top.

File: web/blog/tenzir-platform-v1.5/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+title: "Tenzir Platform v1.5: Revamped Dashboards"
+slug: tenzir-platform-v1.5
+authors: [dit7ya, gitryder]
+date: 2024-12-04
```
Comment (mavam) [inline web/blog/tenzir-platform-v1.5/index.md:5]: Needs adjustment

Comment (dominiklohmann) [reply web/blog/tenzir-platform-v1.5/index.md:5]: Good call.

```suggestion
date: 2024-12-06
```

**Pull request merged**